files/KoreBuild/scripts/dotnet-install.ps1

```diff
@@ -501,89 +498,109 @@ function Prepend-Sdk-InstallRoot-To-Path([string]$InstallRoot, [string]$BinFolde
     }
 }
 
-$CLIArchitecture = Get-CLIArchitecture-From-Architecture $Architecture
-$SpecificVersion = Get-Specific-Version-From-Version -AzureFeed $AzureFeed -Channel $Channel -Version $Version
-$DownloadLink = Get-Download-Link -AzureFeed $AzureFeed -SpecificVersion $SpecificVersion -CLIArchitecture $CLIArchitecture
-$LegacyDownloadLink = Get-LegacyDownload-Link -AzureFeed $AzureFeed -SpecificVersion $SpecificVersion -CLIArchitecture $CLIArchitecture
+$lockFile = Join-Path $PSScriptRoot "dotnetinstall.lock"
 
-if ($DryRun) {
-    Say "Payload URLs:"
-    Say "Primary - $DownloadLink"
-    if ($LegacyDownloadLink) {
-        Say "Legacy - $LegacyDownloadLink"
-    }
-    Say "Repeatable invocation: .\$($MyInvocation.Line)"
-    exit 0
+$waitTime = 0
+while((Test-Path $lockFile) -and ($waitTime -lt 120))
+{
+    Start-Sleep -Seconds 10
```

Comment:
Before sleeping, it might be useful to log a message such as "Another installation of .NET Core is in process. Waiting for that installation to complete..."